Cargo.toml

```diff
@@ -490,8 +490,16 @@ sha1 = { version = "0.10.6", features = ["std"] }
 tempfile = { workspace = true }
 time = { workspace = true, features = ["local-offset"] }
 unindent = "0.2.3"
-uucore = { workspace = true, features = ["entries", "process", "signals"] }
+uucore = { workspace = true, features = [
+  "entries",
+  "process",
+  "signals",
+  "utmpx",
+] }
+serde = { version = "1.0.202", features = ["derive"] }
```

Comment:
do we really need 3 new dependencies ?

Reply:
We don't need them for building , but i am basically serializing utmp records into bytes and writing it to a file to test the ``uptime`` on a file instead of just shipping binary files into the repo for testing, leading to much more easily auditable code. I could try to write a serializer to turn my struct into binary instead of using dependencies but i believe it would require some research into what byte order different architectures use.

Reply:
sorry, i missed that it was in [dev-dependencies]
sorry